ui/perfherder/Navigation.jsx

```diff
@@ -17,7 +17,10 @@ const Navigation = ({ user, setUser, notify }) => (
         </Link>
       </NavItem>
       <NavItem>
-        <Link to="./comparechooser" className="nav-link btn-view-nav">
+        <Link
+          to="./comparechooser?checkboxSelected=true"
```

Comment:
What does this query param do?

Reply:
Dave Hunt stated in a comment to 'remove the checkbox and only allow comparing specific revisions', this is why `checkboxSelected` query param is mandatory now: Initially by default revisions were only needed for the 'New' component dropdown and revisions for the 'Base' component were fetched only as needed (in order to fetch the revisions for the 'Base' component the `checkboxSelected` query param needs to be provided).

Reply:
Here's the piece of code I am referring to from the `SelectorCard` Component:
 `if (this.state.checkboxSelected || revisionState === 'newRevision') {
      this.fetchRevisions(selectedRepo);
}`
In the case of the 'Base' component dropdown revisionState equals 'originalRevision' and it won't fetch revisions unless `checkboxSelected` is provided.

Reply:
I see what you mean. I think it'd be better to refactor the `SelectorCard` component to remove the `checkboxSelected` condition (and the query param) since it's no longer needed.